src/probe.ts

```
@@ -0,0 +1,116 @@
+import { stdin, stdout } from 'node:process'
```

ESLint

[error] 'node:process' imported multiple times.

(import/no-duplicates)